crypto/s2n_drbg.c

```diff
@@ -129,8 +131,12 @@ int s2n_drbg_instantiate(struct s2n_drbg *drbg, struct s2n_blob *personalization
     memcpy_check(ps.data, personalization_string->data, MIN(ps.size, personalization_string->size));
 
     /* Seed / update the DRBG */
-    GUARD(s2n_drbg_seed(drbg));
+    GUARD(s2n_drbg_seed(drbg, &ps));
 
+    /* After initial seeding, pivot to RDRAND if available and not overridden */
+    if (drbg->entropy_generator == NULL && s2n_cpu_supports_rdrand()) {
+        drbg->entropy_generator = s2n_get_rdrand_data;
+    }
```

Comment:
If s2n_cpu_supports_rdrand() isn't true, we'll end up having no PR but also no reseeding at all, even past the usage limit. Should we plug in get urandom in that case? It'll be slow but safe.